### DISCUSSION TITLE
我的账号被封了，登录不了

### DISCUSSION THREAD

**Discussion by xbingbo** [community/community#141780] [Other Feature Feedback, Questions, & Ideas] [answered]
你好我是github中国用户，我的账号ibingbo.zh@gmail.com不能登录了，如下图所示：
![image](https://github.com/user-attachments/assets/b72bbd31-3655-4728-87d8-b85dd36c2851)
请帮忙解决下，谢谢

**Comment by queenofcorgis** [comment, marked as answer]
Hi @xbingbo, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note: We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.
